### PR TITLE
Add action feedback for toggle delta solo

### LINKED ISSUE #584: Is it possible to vocalize when activating or deactivating the Delta solo from the action?
Hello,
there is action to activate or deactivate the Delta solo, is it possible for OSARA to send information to the screen reader?
The action is:
FX: Toggle delta solo for last focused FX
Very convenient, as for the bypass it would also be useful for the delta solo.
Greetings!

### PR DESCRIPTION
**Pull request by leonardder**
Fixes #584

Note: nothing needed around fx param ident stuff. Also, as this action is supported in REAPER 6.37 and above, do we need to bump the minimum version?

### CODE REVIEW

**Comment by Lo-lo78** [comment]
It's ok Leonard!

👍

File: src/reaper_osara.cpp [hunk 1]
```diff
@@ -868,6 +868,12 @@ void postToggleAllTracksFxBypass(int command) {
 		translate("all tracks FX active"));
 }
 
+void postToggleLastFocusedFxDeltaSolo(int command) {
+	outputMessage(GetToggleCommandState(command) ?
+		translate("delta solo on") :
+		translate("delta solo off"));
```
Comment (jcsteh) [inline src/reaper_osara.cpp:874]: Given the length of the words "delta solo", what do you think about "delta on"/"delta off"? Alternatively, "enabled delta solo"/"disabled delta solo"? My aim here is to make sure the users get the important information (the on/off state) as early as possible.

**Review by jcsteh — COMMENTED** [review]
Thanks!

I don't think we need to bump the minimum version. This won't break anything on older versions; the functionality just can't be triggered. I usually only bump the minimum version when something would break on an older version.

File: src/reaper_osara.cpp (hunk 1 above)
Comment (jcsteh) [reply src/reaper_osara.cpp:874]: Or "soloed delta" and "unsoloed delta", but that seems weird.

Comment (leonardder) [reply src/reaper_osara.cpp:874]: I think Enabled/disabled delta solo is ok. The word solo seems pretty relevant in this context.

**Review by jcsteh — APPROVED** [review]

**Pull request merged**